install/install.sh

```diff
@@ -13,7 +13,17 @@
 
 export PATH=$PATH:$ANSIBLE_HOME/bin
 PLAYBOOKFILE="qpc_playbook.yml"
-
+RELEASE_TAG='-e RELEASE_TAG=0.0.43'
+POSTGRES_VERSION='-e POSTGRES_VERSION=9.6.10'
+#CLI_PACKAGE_VERSION='-e CLI_DEFAULT_VERSION=qpc'
+#TODO: Uncomment CLI_PACKAGE_VERSION when cutting a release
+#Additionally you will need to make sure that the variables is in args
```

Comment:
This is a suggestion, not a blocker, but move this `#TODO` above the line that needs to be uncommented. Also, possibly move the `#Additionally` to right above line 24 and change it to a `#TODO` as well. 

Also, a nitpick change `make sure that the variables is in args` to `make sure that the variables are in args`.